eng/Directory.Build.Common.props

```diff
@@ -93,6 +93,8 @@
 
   <PropertyGroup Condition="'$(IsShippingClientLibrary)' == 'true' and '$(TF_BUILD)' == 'true'">
     <ContinuousIntegrationBuild>true</ContinuousIntegrationBuild>
+    <!-- See https://github.com/dotnet/sourcelink/tree/main/docs#deterministicsourcepaths for details. -->
+    <DeterministicSourcePaths>true</DeterministicSourcePaths>
```

Comment:
This seems unrelated. Is there a reason for this update?

Reply:
It's related but I'm definitely going to make sure the right thing keeps happening. That block I conditioned on < netcoreapp3.1? The point is to make symbols work for nupkgs but >= netcoreapp3.1 does the right thing so long as `DeterministicSourcePaths` is true. According to docs, as long as `ContinuousIntegrationBuild` and `Deterministic` is set, this property gets set to `true`. While I know `ContinuousIntegrationBuild` gets set to `true` in our CIs, I can't find for sure what sets `Deterministic` to true and want to make sure, now that I'm conditioning this older block, it's always set correctly.